proto/buf/alpha/registry/v1alpha1/webhook.proto

```diff
@@ -104,6 +104,6 @@ message Webhook {
   string repository_name = 5;
   // The webhook repository owner name.
   string owner_name = 6;
-  // The subscriber's callback URL where notifications are delivered.
-  string callback_url = 7;
+  // The subscriber's callback host where notifications are delivered.
+  string callback_host = 7;
```

Comment:
`host` typically refers to only the domain part of a url, eg for `https://example.com/route/here` only `example.com` is the host.

If we want to support users supplying `https://example.com/route/here` to call `https://example.com/route/here/buf.alpha.webhook.v1alpha1.EventService/Event`, something like `callback_url_prefix` seems more appropriate (and in the comment explain that we will add the `pkg.service/method` to it)

Reply:
That is fair. You know -- what if we kept the previous name and I'll add some better doc strings around what specifically this can be?

Reply:
I think it's much clearer to ask for the complete `callback_url`, and for the time that we're only supporting `connect` backends, we validate the `/buf.alpha.webhook.v1alpha1.EventService/Event` suffix.

Later, when/if we support JSON or other protos, we can validate that suffix conditionally based on a new param. And still, that field will hold true to itself, the complete callback url.

Reply:
The URL doesn't strictly specify encoding, `buf.alpha.webhook.v1alpha1.EventService/Event` could be called with either json or proto payloads.